lib/Raven/Client.php

```diff
@@ -269,9 +269,8 @@ public function install()
         $this->error_handler->registerErrorHandler();
         $this->error_handler->registerShutdownFunction();
 
-        if ($this->_curl_handler) {
-            $this->_curl_handler->registerShutdownFunction();
-        }
+        // Register final shutdown function to send fatal error via curl async.
+        $this->registerShutdownFunction();
```

Comment:
It's not necessary, but the idea here was that this would be the suggested way to register shutdown function - given that _curl_handler is protected, this line is useful for anyone who is writing custom code and wants to reproduce the functionality of the install() method.